src/server/game/Spells/SpellMgr.cpp

```diff
@@ -7082,10 +7082,15 @@ void SpellMgr::LoadDbcDataCorrections()
         spellInfo->SpellLevel = 0;
     });
 
-    // Drain Life - Bryntroll
+    // Drain Life - Bryntroll & Death Coil Warlock
```

Comment:
You can remove this comment as there are more spells here now with the description after the spells :)

Reply:
done 👍